$NetBSD: patch-ac,v 1.11 2019/02/21 01:37:14 tsutsui Exp $

- make ghostscript support optional
- use pkgsrc ${PREFIX} for ghostscript files

--- config.h.orig	2019-02-17 16:07:37.145241693 +0000
+++ config.h
@@ -112,10 +112,12 @@
  * should not need to be changed
  */
 
+#ifdef DOGS
 /* #define GS_PATH "/usr/local/bin/gs" */
-#define GS_PATH "gs"
-/* #define GS_LIB  "."                 */
-/* #define GS_DEV  "ppmraw"            */
+#define GS_PATH "@PREFIX@/bin/gs"
+#define GS_LIB "@PREFIX@/share/ghostscript"
+#define GS_DEV "ppmraw"
+#endif
 
 
 /***************************************************************************
